.github/workflows/hubble-cli.yaml

```diff
@@ -0,0 +1,40 @@
+name: Hubble CLI tests
+
+on:
+  pull_request: {}
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+jobs:
+  check_changes:
+    name: Deduce required tests from code changes
+    runs-on: ubuntu-22.04
+    outputs:
+      tested: ${{ steps.tested-tree.outputs.src }}
+    steps:
+      - name: Check code changes
+        uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36 # v3.0.2
+        id: tested-tree
+        with:
+          filters: |
+            src:
+              - '!(test|Documentation)/**'
+
+  build-hubble-cli-release-binaries:
```

Comment:
It's not mentioned anywhere, but I assume the Hubble CLI unit tests are run as part of the regular `make unit-tests` CI workflow?

Reply:
it runs as a part of `ci-integration`: https://github.com/cilium/cilium/actions/runs/8723948645/job/23933377817#step:13:237